src/public/js/export-configuration.js

```diff
@@ -21,8 +22,11 @@ function getExportConfiguration() {
         exportPath: $('#output').val(),
         exportFormat: $('#format').val()
     };
-
-    ipcRenderer.send('export-tags', exportConfig); 
+    if(exportConfig.exportFormat == "tfrecord") {
```

Comment:
I try to get in the habit of doing `===` in JS -- it can help avoid some heartache/weirdness down the road. And if you're using `"tfrecord"` in more than one place, I'll create a constant for it somewhere and reference that instead of a literal.